tools/webidl_binder.py

```diff
@@ -525,6 +525,11 @@ def render_function(class_name, func_name, sigs, return_type, non_pointer, copy,
       return_postfix += ', &temp)'
 
     c_return_type = type_to_c(return_type)
+
+    for name, enum in enums.items():
+        if c_return_type == name:
```

Comment:
should be 2 spaces, not 4

Reply:
instead of doing this loop here, how about adding a line to `type_to_c` which does
```
elif t in enums:
  # enums are just integers in C
  ret = 'int'
```
and we can call `type_to_c` here?